src/Experimental/Edit.NET/ViewModels/AppViewModel.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.IO;
+using System.Text.Json;
+using Avalonia.Styling;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Consolonia.Controls;
+using Edit.NET.DataModels;
+using TextMateSharp.Grammars;
+
+namespace Edit.NET.ViewModels
+{
+    public partial class AppViewModel : ObservableObject
+    {
+        private static readonly string SettingsFilePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Edit.NET", "settings.json");
+
+        public AppViewModel()
+        {
+
+        }
+
+        [ObservableProperty]
+        private ConsoloniaTheme _uITheme = NET.ConsoloniaTheme.Modern;
+
+        [ObservableProperty]
+        private ThemeVariant _uIThemeVariant = ThemeVariant.Dark;
+
+        [ObservableProperty]
+        private bool _showTabs = true;
```

Comment:
It looks like these properties are not used anyhow. Need to review layers of data